### PR TITLE
Bahubali cast details

### PR DESCRIPTION
**Pull request by uiuxmathesh**
# Bahubali Cast list
Added all the cast details of bahubali

### CODE REVIEW

File: cast.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Prabhas
+Rana
+Rohit shetty
```
Comment (ragavkumarv) [inline cast.txt:3]: Add Category of people

File: cast.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+Anushka
+Thamannah
```
Comment (ragavkumarv) [inline cast.txt:5]: Add character name

**Review by ragavkumarv — CHANGES_REQUESTED** [review]
Please address the below comments

File: cast.txt (hunk 1 above)
Comment (uiuxmathesh) [reply cast.txt]: Resolved: Added Categories

**Review by ragavkumarv — APPROVED** [review]
Looks good. Good job buddy. LGTM. 🎉

**Pull request merged**